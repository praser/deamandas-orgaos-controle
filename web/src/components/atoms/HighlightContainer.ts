import styled from 'styled-components';

const HighlightContainer = styled.div`
  /* Layout */
  box-sizing: border-box;
  display: flex;
  flex-direction: row;
  align-items: center;
  min-height: 102px;
  padding: 20px 28px;
  position: relative;
  width: 100%;

  /* Appearance */
  color: rgb(133, 135, 150);
  background-clip: border-box;
  background-color: rgb(255, 255, 255);
  border-color: rgb(227, 230, 240);
  border-style: solid;
  border-width: 1px;
  border-left-color: ${({ color }) => color || 'rgb(78, 115, 223)'};
  border-left-style: solid;
  border-left-width: 4px;
  box-shadow: rgba(58, 59, 69, 0.15) 0px 2.4px 28px 0px;

  /* Other */
  border-radius: 5.6px;
  overflow-wrap: break-word;
`;

export default HighlightContainer;
