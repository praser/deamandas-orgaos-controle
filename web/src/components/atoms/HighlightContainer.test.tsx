import React from 'react';
import { render, screen } from '@testing-library/react';
import HighlightContainer from './HighlightContainer';

describe('HighlightContainer component', () => {
  let highlightcontainer: HTMLElement;
  const testId = 'testid';

  beforeEach(() => {
    render(<HighlightContainer data-testid={testId} />);
    highlightcontainer = screen.getByTestId(testId);
  });

  it('is expected to match the snapshot', () => {
    expect(highlightcontainer).toMatchSnapshot();
  });
});
