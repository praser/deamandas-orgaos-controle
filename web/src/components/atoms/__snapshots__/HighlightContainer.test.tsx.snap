// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HighlightContainer component is expected to match the snapshot 1`] = `
<div
  class="sc-bdvvaa iPgzyR"
  data-testid="testid"
/>
`;
